views/lib/forms/paginatedForm.hbs

```diff
@@ -7,7 +7,7 @@
 
 	<div class="stages">
 		{{#for 1 (add sections 1) 1}}
-			<label role="button" class="section-button" aria-pressed="false" tabindex="0" for="section-{{this}}">{{this}}</label>
+			<label role="button" class="section-button" aria-pressed="false" tabindex="0" for="section-{{this}}" data-testid="sectionButton-{{this}}">{{this}} </label>
```

Comment:
please be more consistent with styling here, e.g. `section-button-{{this}}`

Reply:
Updated it.